Core/Reporting.cpp

```diff
@@ -458,8 +458,6 @@ namespace Reporting
 	bool IsSupported()
 	{
 		// Disabled when using certain hacks, because they make for poor reports.
-		if (g_Config.bTimerHack)
```

Comment:
Oops, not meant for this PR.

-[Unknown]

Reply:
thanks, moved to the other PR.